R/get.R

```diff
@@ -5,6 +5,13 @@
 #' @param prefix Filters users by prefix (username, first name, or last name).
 #' The filter is case insensitive.
 #' @param limit The max number of records to return
+#' @param user_role Optionally filter by user role ("administrator",
+#' "publisher", "viewer"). Pass in a vector of multiple roles to match any value
+#' (boolean OR). When `NULL` (the default), results are not filtered.
```

Comment:
```suggestion
#' (boolean OR). When `NULL` (the default), all results are returned.
```

Feel free to take or leave this suggestion, but is this true? That not filtered == all results? I believe yes for sure for `user_role`, but not sure about `account_status` are locked/inactive accounts included if you don't specify anything?

As for the suggestion of "all results" vs "results not filtered" either is fine, the first seems slightly more approachable to me, but I don't super strongly hold that.

Reply:
Yep, this is true! The default for `account_status` is to not filter by status when nothing's passed in — this was my understanding from reading the API docs, and I just confirmed it experimentally (by filtering for all types, which uses boolean OR, and comparing that to unfiltered results).

I agree that the current wording isn't a paragon of accessibility, but to my ear the suggested change sounds almost tautological, in the sense that "result" reads to me as meaning "the set of accounts returned by the function/API call. I wonder if this sentence is even needed — to me, the "Optionally" at the start of the parameter definition says everything that this sentence is trying to say (i.e. this is an optional parameter, and if you don't pass anything, the filtering operation described doesn't happen).

Reply:
Yeah, I can see arguments for both ways. I do think we should mention the default, it's not good to not say what the default is. Like I said, feel free to reject the suggestion — maybe leaning heavily on `filter` and what that means is enough here.